virt/virtual_machines/creating_vms_rh/virt-creating-vms-from-rh-images-overview.adoc

```diff
@@ -15,7 +15,10 @@ Cluster administrators can enable automatic subscription for {op-system-base-ful
 You can create virtual machines (VMs) from operating system images provided by Red Hat by using one of the following methods:
 
 * xref:../../../virt/virtual_machines/creating_vms_rh/virt-creating-vms-from-templates.adoc#virt-creating-vms-from-templates[Creating a VM from a template by using the web console]
+// Hiding in ROSA/OSD, not supported
+ifndef::openshift-rosa,openshift-dedicated[]
```

Comment:
In the previous file (virt/virtual_machines/creating_vms_rh/virt-creating-vms-from-instance-types.adoc) it seemed that instance types were supported if you were running OCP Virt 4.15+. Should this be hidden?

Reply:
I think that is right. I marked this up well before 4.15.